state-chain/pallets/cf-ingress-egress/src/lib.rs

```diff
@@ -2689,13 +2690,17 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 		origin: &DepositOrigin<T, I>,
 	) -> AmountAndFeesWithheld<T, I> {
 		if matches!(origin, &DepositOrigin::DepositChannel { .. }) {
-			Self::withhold_ingress_or_egress_fee(IngressOrEgress::Ingress, asset, available_amount)
+			Self::withhold_ingress_or_egress_fee(
```

Comment:
nit: i think it better if we write this using a normal match statement (`match origin {
	Deposit {..} => .., Vault {..} => .., }`) in case we add a new variant and then forget to handle it correctly.